appveyor.yml

```diff
@@ -4,7 +4,6 @@ build_script:
   - cmd: build.cmd CleanInternetCaches
   - cmd: build.cmd AppVeyorBuildVersion
   - cmd: build.cmd All
-  - cmd: build.cmd SourceLink
```

Comment:
@baronfel , didn't you fix SourceLink on the last commit?

Reply:
Or does it always run now without a separate command?

Reply:
It was removed separately to this PR, this flushed some more stuff from build.fsx

Reply:
Yes, though I did add it back in using the new clitool and msbuild tasks that @ctaggart helped me get working. It now happens transparently during the build, due to the presence of the `SourceLinkCreate` msbuild property.

There's a separate testing step that happens to verify the sourcelinking occurred correctly.